engines/chewy/ngshext.h

```diff
@@ -0,0 +1,48 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef CHEWY_NGSHEXT_H
+#define CHEWY_NGSHEXT_H
+
+#include "chewy/memory.h"
+#include "chewy/mcga.h"
+#include "chewy/mouse.h"
+
+#include "chewy/io_game.h"
+#include "chewy/cursor.h"
+#include "chewy/flic.h"
+#ifdef AIL
+#include "chewy/sound_player.h"
+#endif
+
+namespace Chewy {
+
+#define MAXPFAD 81
+
+class sblaster;
+
+extern sblaster *snd;
```

Comment:
Agreed. I've removed everything from it apart from the #includes. I briefly experimented with removing the file entirely, but in the end decided it was better to have the file and includes rather than trying to replicate them in every file that included that header.